### DISCUSSION TITLE
Bulk import / creation of Expense accounts ?

### DISCUSSION THREAD

**Discussion by nodecentral** [firefly-iii/firefly-iii#6795] [Firefly III] [answered]
I’m just starting out and working on getting all my content together..

I can see within my exported bank statements (.csv) , all the companies I have made payments to recently (and there are a lot)

To save me creating an account for each one manually, is there a way for me to do a bulk import of them, so they are automatically set up, ready to be used ?

**Comment by JC5** [comment, marked as answer]
You don't need to, if they don't exist they will be created.

**Reply by nodecentral** [reply]
That’s great, many thanks @JC5 

When it comes to the import itself, as the companies (expenses accounts) I’ve used, all seem to be recorded in capital letters on my statement, does Firefly iii care about the case of the text ?  Or will it just do a close as possible string match ?

Oh and on the import mapping, is it `Opposing Account (name)` I should use to create the new expense accounts. ?

**Reply by JC5** [reply]
I'm not entirely sure but I believe it's case insensitive. The mapping you mention is indeed the correct one.

**Reply by nodecentral** [reply]
Many thanks @JC5
